tests/admin_views/test_autocomplete_view.py

```diff
@@ -185,20 +184,27 @@ def test_select(self):
         # Load next page of results by scrolling to the bottom of the list.
         for _ in range(len(results)):
             search.send_keys(Keys.ARROW_DOWN)
```

Comment:
So, here... is there something we can wait on to say that the http request completed before we move on to the asserts below?